src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -128,6 +129,34 @@ public boolean link() throws IOException, InterruptedException {
             return false;
         }
 
+        // Prepare files
+        String sourceOS = projectConfiguration.getTargetTriplet().getOs();
+        Path rootPath = paths.getSourcePath().resolve(sourceOS);
+        Path userManifest = rootPath.resolve(Constants.MANIFEST_FILE);
+        Path androidPath;
+        if (!Files.exists(userManifest)) {
+            // copy manifest and assets to gensrc/android
+            androidPath = paths.getGenPath().resolve(sourceOS);
+            Path genManifest = androidPath.resolve(Constants.MANIFEST_FILE);
+            Logger.logDebug("Copy " + Constants.MANIFEST_FILE + " to " + genManifest.toString());
+            FileOps.copyResource("/native/android/AndroidManifest.xml", genManifest);
+            FileOps.replaceInFile(genManifest, "package='com.gluonhq.helloandroid'", "package='" + projectConfiguration.getAppId() + "'");
+            FileOps.replaceInFile(genManifest, "A HelloGraal", projectConfiguration.getAppName());
+
+            Path androidResources = androidPath.resolve("res");
```

Comment:
Shouldn't this be `.resolve("assets").resolve("res")`?

Reply:
No, I've removed that extra `assets` folder. In fact you can have android/res and android/assets folders (the latter will have to be included in the package command too)

Reply:
With that in mind, the latter will then have a default in substrate as `/native/android/assets/assets`? Or is this not really mandatory for the default case?

Reply:
Yes, we don't need that for the default case.

Reply:
Good.